mobile-save-for-later/src/main/scala/com/gu/sfl/controller/SaveArticlesController.scala

```diff
@@ -52,6 +53,10 @@ class SaveArticlesController(updateSavedArticles: UpdateSavedArticles)(implicit
           case i: IdentityServiceError =>  identityErrorResponse
           case m: MissingAccessTokenError => missingAccessTokenResponse
           case u: UserNotFoundError => missingUserResponse
+          case OktaOauthValidationError(e, InvalidOrExpiredToken) => oktaOauthError(e, StatusCodes.unauthorized)
+          case OktaOauthValidationError(e, MissingRequiredClaim(_)) => oktaOauthError(e, StatusCodes.forbidden)
```

Comment:
Would this error suggest a misconfiguration rather than a lack of permission? If the request has the right scopes it should also have the right claims.